text/0000-pack-toml.md

```diff
@@ -0,0 +1,123 @@
+# Meta
+[meta]: #meta
+- Name: Support for `pack.toml`
+- Start Date: 2021-11-16
+- Author(s): [jkutner](http://github.com/jkutner)
+- Status: Draft
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+This is a proposal for a `pack.toml` file that is a special case of the `project.toml` descriptor file. The `pack.toml` will include configuration options specific to the Pack CLI, which may not be honored by other platforms.
+
+# Definitions
+[definitions]: #definitions
+
+- *project descriptor* - a extension spec defining the [`project.toml`](https://github.com/buildpacks/spec/blob/main/extensions/project-descriptor.md) file.
+
+
+# Motivation
+[motivation]: #motivation
+
+The `project.toml` file has been a useful and essential tool for buildpack users who need to codify certain arguments of their builds. However, there has been much debate around how to handle configuration options that may not be supported by all platforms.
+
+We desire a descriptor file that is:
+* specific to pack (i.e. not platform-neutral)
+* advertises that the repo containing the file can be built with pack
+* adheres to the best practices and learnings from [`project.toml`](https://github.com/buildpacks/spec/blob/main/extensions/project-descriptor.md)
+
+# What it is
+[what-it-is]: #what-it-is
+
+The Pack CLI will support a new descriptor file, `pack.toml`, that is a superset of the project descriptor.
+* An app MAY have a `pack.toml`
+* An app MAY have a `project.toml`
+* An app MAY have both a `pack.toml` and a `projec.toml`, in which case they will be merged.
```

Comment:
The more I think about it, the more I feel like we shouldn't merge the contents. It's hard to implement things like "removals", "resets", and "append" vs "override". Additionally, by not doing a merging them I feel like it would also be easier for users to comprehend how they are applied. For example, "If `pack.toml` is present it uses that; otherwise it looks for a `project.toml`."